RecoEgamma/ElectronIdentification/python/Identification/heepElectronID_HEEPV70_cff.py

```diff
@@ -98,6 +98,9 @@
 # 3) update the MD5 sum strings below and uncomment the lines again.
 #
 
-central_id_registry.register(heepElectronID_HEEPV70.idName,"45d49ea5f46f3f13f579d208e5e3412b")
+#in 9X, the number of saturated crystals is in the electron so I updated the showershape cuts
+#to use that instead, its slightly differently defined but for our intents and purposes is fine
+#this meant changing the md5sum though, hence why its changed from 45d49ea5f46f3f13f579d208e5e3412b
+central_id_registry.register(heepElectronID_HEEPV70.idName,"49b6b60e9f16727f241eb34b9d345a8f")
```

Comment:
I don't find the comments relevant for the code.
It's fine to have it in the commit message

Reply:
The idea was that somebody may notice that their md5sum id had changed even though the ID is the same. Therefore I put a note why it changed if they went looking for the new one. Anyways removed